tools/buildtools/ambertools/tleap.xml

```diff
@@ -0,0 +1,2043 @@
+<tool id="tleap" name="Build tLEaP" version="0.0.1" >
+    <description>interactively build and run tLEaP files to set up systems with AmberTools</description>
+     <macros>
+        <import>macros.xml</import>
+        <token name="@GALAXY_VERSION@">0</token>
```

Comment:
token is not used?

Should the version be taken from the macro?

Reply:
corrected